tests/test_splitter.py

```diff
@@ -0,0 +1,388 @@
+import numpy as np
+
+from unittest import TestCase
+
+from datumaro.components.project import Dataset
+from datumaro.components.extractor import (DatasetItem,
+    Label, LabelCategories, AnnotationType, Bbox)
+
+import datumaro.plugins.splitter as splitter
+from datumaro.components.operations import compute_ann_statistics
+
+class SplitterTest(TestCase):
+    def _generate_dataset(self, config):
+        # counts = {(0,0):20, (0,1):20, (0,2):30, (1,0):20, (1,1):10, (1,2):20}
+        # attr1 = ['attr1', 'attr2']
+        # attr2 = ['attr1', 'attr3']
+        # config = { "label1": { "attrs": attr1, "counts": counts },
+        #            "label2": { "attrs": attr2, "counts": counts }}
+        def _get_subset():
+            return np.random.choice(['', 'a', 'b'], p = [0.5, 0.3, 0.2])
```

Comment:
It is better to avoid randomness in tests.

Reply:
I'll consider a different way to do this.